superduperdb/backends/base/query.py

```diff
@@ -140,7 +147,7 @@ def _get_flavour(self):
             )
 
     def _get_parent(self):
-        return self.db.databackend.get_table_or_collection(self.identifier)
+        return self.db.databackend.get_table_or_collection(self.table)
```

Comment:
Given that the function is called `get_table_or_collection` it is somewhat counterintuitive to use `self.table` -- why not `self.collection` ?

I think `self.identifier` would make more sense

Reply:
@blythed